controllers/articles-controller.js

```diff
@@ -24,4 +24,12 @@ function patchArticleVoteCount(request, response, next){
     })
 }
 
-module.exports = { getArticleById, getAllArticles, patchArticleVoteCount }
\ No newline at end of file
+function postNewArticle(request, response, next){
+    uploadNewArticle(request.body).then((postedComment) => {
```

Comment:
maybe destructuring then passing arguments to the model function would be cleaner?
e.g.

const {body} = request 
 uploadNewArticle(body) etc.